FreeRTOS-Plus/Demo/AWS/Fleet_Provisioning_Windows_Simulator/Fleet_Provisioning_With_CSR_Demo/DemoSetup/convert_credentials_to_der.py

```diff
@@ -6,8 +6,8 @@
 from cryptography.hazmat.backends import default_backend
 from cryptography.hazmat.primitives import serialization
 
-KEY_OUT_NAME = f"{os.getcwd()}\\corePKCS11_Claim_Key.dat"
-CERT_OUT_NAME = f"{os.getcwd()}\\corePKCS11_Claim_Certificate.dat"
+KEY_OUT_NAME = os.path.join(os.getcwd(), 'corePKCS11_Claim_Key.dat')
+CERT_OUT_NAME = os.path.join(os.getcwd(), 'corePKCS11_Claim_Certificate.dat')
```

Comment:
Currently these scripts only work on windows command line (or powershell), slight tweak to make them work with unix terminals as well for developers who use WSL.